### ISSUE TITLE
Add an option to add regex to ignore locations.

### ISSUE THREAD

**Issue by MagnumDingusEdu** [milesmcc/shynet#82]
It would be really great to have regex filters similar to Hide specific referrers for URL / location, as there are websites where tokens are appended onto the end, and it makes the list of locations insanely long.

![image](https://user-images.githubusercontent.com/55412084/92995343-65914800-f520-11ea-9928-0601d3f6eb07.png)

**Comment by milesmcc** [comment]
Yes, this could definitely be helpful. Adding it to the roadmap.

Could also be helpful to add an option to ignore query parameters.
